db/migrate/20180928143747_add_required_to_user_names.rb

```diff
@@ -0,0 +1,6 @@
+class AddRequiredToUserNames < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null(:users, :first_name, false)
```

Comment:
You should have this from your last PR.

Reply:
For some reason, I couldn't save those changes when I did a rebase. I'm gonna change it manually now.